app/src/main/java/com/opsat7373/moviestars/data/model/Movie.kt

```diff
@@ -1,8 +1,12 @@
 package com.opsat7373.moviestars.data.model
 
 data class Movie(
+    val id : Int,
     val original_title : String,
     val poster_path : String,
-    val vote_average : String) {
+    val vote_average : String,
```

Comment:
don't use underscore in fields names. 
if you need it to map JSON to object, add proper name using camel case and add @SerializedName("field_name")